swing-lib/src/main/java/org/triplea/swing/JButtonBuilder.java

```diff
@@ -32,11 +32,7 @@ public JButtonBuilder(final String title) {
     this.title = title;
   }
 
-  private JButtonBuilder() {}
-
-  public static JButtonBuilder builder() {
-    return new JButtonBuilder();
-  }
+  public JButtonBuilder() {}
```

Comment:
Unless there's another constructor in this class having a no-op public constructor is redundant

Reply:
There is another constructor above. The position of this constructor is not terribly standard though, usually no-arg ones come before.

Updated ordering: cfac03cf4